pkg/server/methods/methods.go

```diff
@@ -8,7 +8,8 @@ import (
 )
 
 type Methods struct {
-	Ctx   context.Context
-	Conn  jsonrpc2.Conn
-	Cache *utils.Cache
+	Ctx       context.Context
+	Conn      jsonrpc2.Conn
+	Cache     *utils.Cache
+	LsContext *utils.LsContext
```

Comment:
We can call it `Context` without the Ls at the beginning since we're already in the ls haha
same for the type imo
wdyt?